dbfit-java/sqlserver/src/integration-test/resources/acceptancescripts-SqlServer.sql

```diff
@@ -82,6 +82,22 @@ SET ANSI_NULLS ON
 GO
 SET QUOTED_IDENTIFIER ON
 GO
+IF NOT EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'[dbo].[Increment_P]') AND type in (N'P', N'PC'))
+BEGIN
+EXEC dbo.sp_executesql @statement = N'CREATE PROCEDURE [dbo].[Increment_P]
+	@counter INT OUTPUT
+AS
+BEGIN
+	SET @counter = isnull(@counter, 1) + 1
```

Comment:
> What about isnull? We have OBJECT_ID() and CAST() elsewhere.

I don't know - for the user-defined functions and procedures we're using `CamelCase`. Maybe SQL Server preserves the case - as the object has been created, does it? (If yes - that may be the general convention). Anyway - in the meanwhile I'll change that to uppercase too.